source/internal.d.ts

```diff
@@ -3,6 +3,8 @@ import type {Simplify} from './simplify';
 import type {Trim} from './trim';
 import type {IsAny} from './is-any';
 
+export type {UnknownRecord} from './unknown-record';
```

Comment:
I exported this type here to prevent breaking changes.

Reply:
It's better to update the files that use this to use the public type instead.

Reply:
@sindresorhus I did it, for example at `merge-deep.d.ts`. But removing this export from this file will introduce a breaking change.
On my private projects, I have many `import { UnknownRecord } from 'type-fest/source/internal.js';` that will get broken due to removing this export.

Reply:
> On my private projects, I have many import { UnknownRecord } from 'type-fest/source/internal.js'; that will get broken due to removing this export.

If you use internal stuff, it's generally on you if something breaks. But I'll allow it. In the next major version of type-fest, it will not be possible to import internal types, so it will not be a problem then.